packages/sdk-react/src/core/theme/app.css

```
@@ -197,3 +198,8 @@
     animation: dot-typing 1.5s infinite linear;
   }
 }
+
+@import '@react-pdf-viewer/core/lib/styles/index.css';
```

Biome (1.9.4)

[error] This @import is in the wrong position.

Any @import rules must precede all other valid at-rules and style rules in a stylesheet (ignoring @charset and @layer), or else the @import rule is invalid.
Consider moving import position.

(lint/correctness/noInvalidPositionAtImportRule)